storagenode/blobstore/filestore/dir.go

```diff
@@ -447,9 +447,10 @@ func (dir *Dir) TryRestoreTrashBlob(ctx context.Context, ref blobstore.BlobRef)
 // EmptyTrash walks the trash files for the given namespace and deletes any
 // file whose mtime is older than trashedBefore. The mtime is modified when
 // Trash is called.
-func (dir *Dir) EmptyTrash(ctx context.Context, namespace []byte, trashedBefore time.Time) (bytesEmptied int64, deletedKeys [][]byte, err error) {
+func (dir *Dir) EmptyTrash(ctx context.Context, namespace []byte, trashedBefore time.Time) (bytesEmptied int64, deletedKeys [][]byte, minPieceTimestamp time.Time, err error) {
```

Comment:
Just as a minor thing, here at the blobstore layer we try to talk about things in terms of _blobs_ and not _pieces_. The blobstore doesn't know what a piece is. Can we change the name of this return value to `minBlobTimestamp` everywhere that it appears under `storagenode/blobstore/`?